src/styles/reset.css

```diff
@@ -0,0 +1,379 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
part of me wants this file to be smaller. like there's some stuff you want, but also lots for IE. I think https://www.joshwcomeau.com/css/custom-css-reset/#the-css-reset is more compact but certainly not as thorough. no action needed, just thinking out loud.